src/middlewared/middlewared/plugins/iscsi.py

```diff
@@ -865,8 +865,7 @@ def extent_naa(self, naa):
                 diskchoices[f'zvol/{snap["name"]}'] = f'{snap["name"]} [ro]'
 
         for disk in await self.middleware.call('disk.get_unused'):
-            size = await self.middleware.call('notifier.humanize_size', disk['size'])
-            diskchoices[disk['name']] = f'{disk["name"]} ({size})'
+            diskchoices[disk['name']] = f'{disk["name"]}|{disk["size"]}'
```

Comment:
We are changing format here, is this intentional ?

Reply:
Yes, its not job of middleware to provide only formatted data to that extent, idk why it was there before.